man/systemd.unit.xml

```diff
@@ -764,12 +764,12 @@
       </varlistentry>
 
       <varlistentry>
-        <term><varname>StartLimitIntervalSec=</varname></term>
-        <term><varname>StartLimitBurst=</varname></term>
+        <term><varname>StartLimitIntervalSec=<replaceable>interval</replaceable></varname></term>
+        <term><varname>StartLimitBurst=<replaceable>burst</replaceable></varname></term>
 
-        <listitem><para>Configure unit start rate limiting. By default, units which are started more than 5 times
-        within 10 seconds are not permitted to start any more times until the 10 second interval ends. With these two
-        options, this rate limiting may be modified. Use <varname>StartLimitIntervalSec=</varname> to configure the
+        <listitem><para>Configure unit start rate limiting. Units which are started more than
+        <replaceable>burst</replaceable> times within <replaceable>interval</replaceable> interval
```

Comment:
should this be "within a <replacable>interval</replaceable> time interval"? or so? i.e. with the extra "a" and the clarification "time interval"?

Reply:
Isn't interval the wrong term anyway? 
The interval between to start attempts is set by RestartSec AFAIK.
The 10s is more like a time window.

Reply:
Updated.

Reply:
> Isn't interval the wrong term anyway?

"Time window" and "time interval" are two interchangeable terms. Time period has different meaning, it's used more when something repeats. I don't think there's any confusion in this context, and using "interval" is appropriate became the option is called so.